spec/components/schemas.yaml

```diff
@@ -644,11 +644,12 @@ components:
         bbox:
           $ref: "./schemas.yaml#/components/schemas/bbox"
         geometry:
+          $ref: "./schemas.yaml#/components/schemas/itemGeometry"
           # we could use the 'original' schema, but it doesn't
           # contain a useful example, hence we use our own
           # schema with appropriate example
           # $ref: https://geojson.org/schema/Geometry.json
-          $ref: "./schemas.yaml#/components/schemas/geometryGeoJSON"
+          # $ref: "./schemas.yaml#/components/schemas/geometryGeoJSON"
```

Comment:
This section will be deleted, if this PR is approved, i.e. if you agree, that all geometry types should be valid options in the specs for all methods on item endpoints.